generators/app/templates/src/app/screens/Registration/constants.tsx

```diff
@@ -0,0 +1,3 @@
+import { stringArrayToObject } from '~utils/array';
+
+export const FIELDS = stringArrayToObject(['name', 'surname', 'email', 'password', 'confirmPassword']);
```

Comment:
I have my concerns about defining this kind of constants, we usually don't use them in more than one place and we are adding a file that may not be necessary.
Don't need to change this because we are used to doing this but i just wanted to make my point.

Reply:
I see your point. I believe that this is still better than having several literals spread across the code since it allows for easier understandability. 
Maybe we could see a way to embed this directly in the component to avoid adding an extra file, but I'm not sure how good that practice is.

Reply:
I'm fine with this constants, but maybe we could have them in the component to avoid a new file. I think we should be more flexible regarding where to put constants. If the component is small, no need to move this to a constant file unless there are several constants.